scripts/scil_compute_fodf.py

```diff
@@ -104,69 +99,25 @@ def main():
                                  args.frf_file])
     assert_outputs_exist(parser, args, arglist)
 
-    nbr_processes = args.nbr_processes
-    parallel = True
-    if nbr_processes is not None:
-        if nbr_processes <= 0:
-            nbr_processes = None
-        elif nbr_processes == 1:
-            parallel = False
-
     full_frf = np.loadtxt(args.frf_file)
-
-    if not full_frf.shape[0] == 4:
-        raise ValueError('FRF file did not contain 4 elements. '
-                         'Invalid or deprecated FRF format')
-
-    frf = full_frf[0:3]
-    mean_b0_val = full_frf[3]
-
     vol = nib.load(args.input)
-    data = vol.get_data()
-
+    data = vol.get_fdata()
```

Comment:
The problem with this is that you load in f64. Maybe use dataobj() ? @jchoude @arnaudbore what do you think about that ?

Reply:
agree with guillaume we should use dataobj()